tools/nl/nodejs_query_differ/README.md

```diff
@@ -0,0 +1,14 @@
+## NodeJs Query Differ
+
+This is a commmand line tool to get the diffs between two different runs of
+the Nodejs Query tests (that are run by cronjobs on gke) and outputs the diffs
+in a json that is saved locally.
+
+To run this:
+
+```
+./run.sh -e <ENV> -b <BASE_FOLDER> -t <TEST_FOLDER>
+```
+
```

Comment:
Add a todo to have a UI for showing the diff result?

Reply:
done